modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java

```diff
@@ -154,7 +258,14 @@ private void _testPostExportTask(
 	@Inject
 	private CompanyLocalService _companyLocalService;
 
+	private ObjectDefinition _objectDefinition1;
```

Comment:
Why do you call it _objectDefinition1 if there is only one? Please rename to _objectDefinition